test/es-module/test-esm-loader-spawn-promisified.mjs

```diff
@@ -0,0 +1,272 @@
+import { spawnPromisified } from '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+import assert from 'node:assert';
+import { execPath } from 'node:process';
+import { describe, it } from 'node:test';
+
+describe('Loader hooks throwing errors', { concurrency: true }, () => {
+  it('throws on nonexistent modules', async () => {
+    const { code, signal, stdout, stderr } = await spawnPromisified(execPath, [
+      '--no-warnings',
+      '--experimental-loader',
+      fixtures.fileURL('/es-module-loaders/hooks-custom.mjs'),
+      '--input-type=module',
+      '--eval',
+      'import \'nonexistent/file.mjs\'',
```

Comment:
Consider using double quotes as I find backslashes don't help with readability. Same for the other backslashes in this file.
```suggestion
      'import "nonexistent/file.mjs"',
```